js-plugins/mcp/src/types.ts

```
@@ -0,0 +1,71 @@
+import type { McpServer } from '@modelcontextprotocol/sdk/server/mcp.js'
+import type { Implementation as McpServerInfo } from '@modelcontextprotocol/sdk/types.js'
+import { Compiler, type Server } from "@farmfe/core"
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)